pygeoapi-config.yml

```diff
@@ -105,6 +105,7 @@ resources:
               href: https://raw.githubusercontent.com/mapserver/mapserver/branch-7-0/msautotest/wxs/data/obs.csv
               hreflang: en-US
         extents:
+            transactions: true
```

Comment:
@alex-mathew @jorgejesus I cannot figure out why the configuration is under the key `extents` that is scoped for the spatial and temporal dimensions. Can you explain what is the design?

Reply:
@francbartoli I talked with Jorge and we agreed on adding a new extensions section under resource and keep transactions it. This section can be used for keeping flags of feature extensions.
```
obs:
       extents:
                  spatial:  . . .
                  temporal: . . .
       extensions:
                  transactions: true
```
What do you think?

Reply:
sounds good to me but let's involve more developers in the decision to get more consensus cc @tomkralidis @justb4 @alpha-beta-soup

Reply:
@francbartoli @tomkralidis  @justb4 @alex-mathew @alpha-beta-soup 

My intial idea was to just to have everything on one place, and anything that would be passed as function arguments, to be under `extents` e.g:

https://github.com/geopython/pygeoapi/blob/d2cbf48cd3f7663caf7520b69451723cfadf47f7/pygeoapi/provider/csv_.py#L140-L141

But  after thinking (nadenken): my current idea is: 
- extents: <dimension data slice, spatial, temporal, pagination, columns>
- extensions: OGC extensions  and functionalities applied to data

Reply:
@jorgejesus @francbartoli Can I move forward with this? (extensions > transactions)